data/api/client-server/relations.yaml

```diff
@@ -0,0 +1,170 @@
+# Copyright 2022 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+swagger: '2.0'
+info:
+  title: "Matrix Client-Server Relations API"
+  version: "1.0.0"
+host: localhost:8008
+schemes:
+  - https
+  - http
+basePath: /_matrix/client/v1
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  $ref: definitions/security.yaml
+paths:
+  "/rooms/{roomId}/relations/{eventId}/{relType}/{eventType}":
+    get:
+      summary: Get the child events for a given parent event, with optional filter.
+      description: |-
+        Retrieve all of the child events for a given parent event, optionally filtering
+        down by relationship type and related event type.
+
+        Note that while this endpoint indicates `relType` and `eventType` being
+        required, they can be individually omitted to retrieve all matching events.
+        When omitting the path parameters, a trailing slash must not be included
+        (otherwise the search will be for an empty string).
+
+        When combining the use of `relType` and `eventType`, the server will return
+        child events which match *both* conditions rather than *either*.
+      operationId: getChildEventsForParentEvent
```

Comment:
"but my image relates to the text I sent after!" - I'd rather maintain the same terminology used in the spec to the operation ID if we can.

Reply:
I don't disagree. It's just that this one is going to be slightly less amenable to the "we'll just clarify it later if we change our minds" approach, since `operationId`s are meant to be constant.

(also, the endpoint itself is called `/relations`, not `/children`).

On a range of -1 to +1, I'm about -0.3 on your name: I'd prefer to see a change but if you want to stick with what you have, that's fine.

Reply:
(and yes, my suggestion isn't great either)

Reply:
your suggestion is a `-0.295` if I were to use your scale, so let's just use that.
```suggestion
      operationId: getRelatingEvents
```